overlord/configstate/configcore/picfg_test.go

```diff
@@ -166,6 +168,94 @@ func (s *piCfgSuite) TestConfigurePiConfigRegression(c *C) {
 	s.checkMockConfig(c, expected)
 }
 
+func (s *piCfgSuite) TestUpdateConfigUC20RunMode(c *C) {
+	restore := release.MockOnClassic(false)
+	defer restore()
+
+	defer func() { dirs.SetRootDir("") }()
```

Comment:
(nitpick) this is not needed, piCfgSuite is using configcoresuite and that sets dirs.SetRootDir() and adds the cleanup too.

Reply:
done